python/ray/serve/_private/api.py

```diff
@@ -110,6 +111,7 @@ def _check_http_options(
 def _start_controller(
     http_options: Union[None, dict, HTTPOptions] = None,
     grpc_options: Union[None, dict, gRPCOptions] = None,
+    logging_config: Union[None, dict, LoggingConfig] = None,
```

Comment:
Can we change this to `system_logging_config` just like how we have `update_system_logging_config` in the client? It's a bit confusing to see `logging_config` in three different places

Reply:
done!

Reply:
Should we also change the config option in `ServeDeploySchema`? There's the same issue of the same option name being used in 3 places. Aware this is an API change, will leave it up to the team to decide.